### PR TITLE
Setup fixes & improvements

### PR DESCRIPTION
**Pull request by kpedro88**
Major change is that setup and init scripts now invoke the Singularity environment automatically. Also generalized the bind mount arguments so the code can be used on non-lpc machines.

Also removed some unneeded items, fixed a couple minor issues, reduced duplication of environment variables.

### CODE REVIEW

**Comment by cmadrid1** [comment]
Yeah this looks good to me. I'll test it later.

File: init.sh [hunk 1]
```diff
@@ -1,6 +1,17 @@
 #!/bin/bash
 
-NAME=coffeaenv
+source local.sh
+NAME=$1
```
Comment (cmadrid1) [inline init.sh:4]: This line breaks our condor bash script here:
https://github.com/cms-svj/t-channel_Analysis/blob/kjp/setup/condor/run_Analyzer_condor.sh#L27
Not sure how to fix it nicely other than to hard code the NAME in this script.

**Comment by cmadrid1** [comment]
Other than that one issue this PR works nicely.

File: init.sh (hunk 1 above)
Comment (cmadrid1) [reply init.sh:4]: The $1 from the run_Analyzer_condor.sh script gets passed into the init.sh script when running on condor, so it doesn't source the environment properly.

Comment (kpedro88) [reply init.sh:4]: I think changing it to `source init.sh ""` will work.

**Pull request merged**